package.json

```diff
@@ -87,6 +87,7 @@
     "karma-mocha": "^0.2.2",
     "karma-mocha-reporter": "^2.0.0",
     "karma-phantomjs-launcher": "^1.0.0",
+    "karma-phantomjs-shim": "github:technologyadvice/karma-phantomjs-shim",
```

Comment:
`karma-phantomjs-shim@1.4.0` published with both `includes` and `repeat`.  Thanks for the contribution.